conda_smithy/lint_recipe.py

```diff
@@ -1,42 +1,43 @@
-# -*- coding: utf-8 -*-
-
-from collections.abc import Sequence, Mapping
-
-str_type = str
-
 import copy
 import fnmatch
-import io
 import itertools
 import json
 import os
 import re
-import requests
 import shutil
 import subprocess
 import sys
+from collections.abc import Mapping, Sequence
 from glob import glob
 from inspect import cleandoc
 from textwrap import indent
 
 import github
+import jsonschema
+import requests
+from conda.exceptions import InvalidVersionSpec
 
 if sys.version_info[:2] < (3, 11):
     import tomli as tomllib
 else:
     import tomllib
 
 from conda.models.version import VersionOrder
+from conda_build.metadata import (
+    FIELDS as _CONDA_BUILD_FIELDS,
+)
 from conda_build.metadata import (
     ensure_valid_license_family,
-    FIELDS as cbfields,
 )
```

Comment:
Why is it splitting these into 2 `import`s from the same module? This seems overly verbose. Is there an option to fix this?

Reply:
I think that's part of the black/isort standard too. `as` imports have their own line 🤷

Reply:
Yes.